pkg/instrumentation/helper.go

```diff
@@ -25,3 +29,31 @@ func IsInitContainerMissing(pod corev1.Pod) bool {
 	}
 	return true
 }
+
+// Check if opentelemetry-auto-instrumentation volume is on the list.
+func IsOtAIVolumeMissing(volumeMounts []corev1.VolumeMount) bool {
+	for _, volumeMount := range volumeMounts {
+		if volumeMount.Name == volumeName {
+			return false
+		}
+	}
+	return true
+}
+
+// Check if EnvVar value contains instrumentation string.
+func IsEnvVarValueInstrumentationMissing(envVar corev1.EnvVar, instrumentation string) bool {
```

Comment:
This is pretty much redundant function the contains on the env var value can be called directly